source/fe/mapping_q_generic.cc

```diff
@@ -1495,8 +1469,8 @@ namespace internal
 
         const unsigned int n_shape_values = data.n_shape_functions;
         const unsigned int n_q_points     = data.shape_info.n_q_points;
-        const unsigned int vec_length     = VectorizedArray<double>::size();
-        const unsigned int n_comp         = 1 + (spacedim - 1) / vec_length;
+        const unsigned int n_lanes        = VectorizedArray<double>::size();
+        const unsigned int n_comp         = 1 + (spacedim - 1) / n_lanes;
         const unsigned int n_hessians     = (dim * (dim + 1)) / 2;
```

Comment:
Could we mark the last three as constexpr?

Reply:
Yes, of course!